### PR TITLE
update K8s req; jenkins-ignore

### PR DESCRIPTION
**Pull request by rosemarymarano**
For 2.2 release, update K8s min requirements, and OpenShift.

### CODE REVIEW

File: docs-source/content/userguide/introduction/introduction.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Detailed instructions are available [here]({{< relref "/userguide/managing-opera
 
 ### Prerequisites
 
-* Kubernetes 1.11.5+, 1.12.3+, and 1.13.0+  (check with `kubectl version`).
+* Kubernetes 1.11.0+, 1.12.3+, and 1.13.0+  (check with `kubectl version`).
```
Comment (russgold) [inline docs-source/content/userguide/introduction/introduction.md]: 1.12.0 is not allowed? I thought we were saying 1.11 and higher was all we required.

Comment (rjeberhard) [reply docs-source/content/userguide/introduction/introduction.md]: Let's say 1.12.0.

Comment (rosemarymarano) [reply docs-source/content/userguide/introduction/introduction.md]: Got it.

Comment (markxnelson) [reply docs-source/content/userguide/introduction/introduction.md]: @rjeberhard I don't agree with this change.  We listed what we tested.  We should not change it without doing the necessary testing.

**Comment by rjeberhard** [comment]
@markxnelson, Pani agreed in the meeting today to do that testing.  I'm happy waiting for the results.  I'll also kick-off some Jenkins runs now at 1.11.0, 1.12.0, etc.

**Pull request merged**